data-access/EditionRepository.cs

```diff
@@ -149,7 +157,10 @@ public async Task ChangeEditionNameAsync(UserInfo user, string name)
             // until the copy process was complete in order to guard against
             // creating an inconsistent copy.
             // What if someone unlocks the source scroll mid-copy?
-            using (var transactionScope = new TransactionScope())
+            using (var transactionScope = new TransactionScope(
+                TransactionScopeOption.Required,
+                new TransactionOptions() { IsolationLevel = System.Transactions.IsolationLevel.ReadUncommitted })
```

Comment:
(I may have left this comment before starting this review, I tried to delete it)
Don't use ReadUncommitted unless you have a *really really* good reason. It's almost always a bad idea. Here you seem to have used it to improve performance - do we have a performance issue when people rename editions? I would drop all ReadUncommitted everywhere. If we are having issues, we'll solve them hopefully without ReadUncommitted.

Reply:
I am very open to getting rid of ReadUncommitted.  The function here is one that creates a copy of an edition (users will use to branch an edition).  So it often makes many writes to the database, 1000s really, so higher isolation levels that lock the whole table when making so many writes cause real deadlock issues.  I was getting a lot of them in the integration tests.  If I could find some other way to prevent locking the table, that would be a nice solution.  I don't want to rely on catching deadlock errors and automatically retrying the query after a brief pause (though some logic for that should be added), because I see no reason to lock the table when these writes are being performed, the permission system in the API gives all the safeguards we need.